cogs/ebay.py

```diff
@@ -0,0 +1,218 @@
+import discord
+from discord.ext import commands
+import aiohttp
+import bs4
+from bs4 import BeautifulSoup as soup
+import datetime
+import re
+import urllib.parse
+import math
+
+headers = {'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36'}
+
+class Ebay(commands.Cog):
+
+    def __init__(self, client):
+        self.client = client
+
+    @commands.command(aliases=['pc','chk','price','pricecheck'])
+    async def check(self, ctx, *, searchTerm):
+        """Gets the average price of an item from Ebay
+        
+        Usage example:
+        !check dell xps 15 -dead -broken -scratched
+        words beginning with `-` are added to the list of words to be filtered out"""
+        if len(' '.join([i for i in searchTerm.split(' ') if not i.startswith('-')])) < 6:
+            embed = await self.error_embed('length')
+            await ctx.send(embed=embed)
+            return
+        async with ctx.channel.typing():
+            filteredTerm, filteredWords = await self.get_filter(searchTerm)
+            page = await self.make_soup(filteredTerm)
+            productList = page.find('ul', {'class': 'srp-results'}).find_all('li', {'class':'s-item'})
+            products = []
+            for i in productList:
+                p = await self.get_product_info(i)
+                if p is not None:
+                    products.append(p)
+
+            def filtered_out(title: str, filteredWords: list):
+                for word in title.split(' '):
+                    if word.lower() in filteredWords:
+                        return True
+                return False
+
+            filteredListings = [i for i in products if filtered_out(i['title'], filteredWords) is False]
+            if len(filteredListings) == 0:
```

Comment:
for what it's worth, empty lists in Python are falsy, so you could rewrite this as:

```suggestion
            if not filteredListings:
```